### DISCUSSION TITLE
TS linter picks up Copilot phantom file

### DISCUSSION THREAD

**Discussion by Aelita4** [community/community#7357] [Copilot Conversations]
When using Copilot, I can request more solutions to the problem with Ctrl+Enter
But, for some reason, linter I'm using (ESLint with TS extension) picks it up and shows many errors in "problems" tab in VS Code
I tried adding "GitHub Copilot" and "Copilot" phrases into ".eslintignore" file and adding them into ".eslintrc.js" like shown in [this guide](https://eslint.org/docs/user-guide/configuring/ignoring-code#ignorepatterns-in-config-files), but nothing changed
Is there any way to exclude that "phantom" file from being picked up by linter?

![image](https://user-images.githubusercontent.com/39732968/140501832-e1eb4d5c-e0cc-4155-9b80-b7b4f2a438de.png)

**Comment by sharunkumar** [comment]
I think this has to be reported to the ESLint extension devs 🤔

**Comment by hepter** [comment]
![image](https://user-images.githubusercontent.com/14141071/143401908-3c172761-ebca-465b-b06a-50e78502a025.png)
I also have the same problem.

**Comment by Exorion1er** [comment]
Any solution today ?

**Comment by hsenag** [comment]
Hi, thanks for reporting this. We're aware of the problem but don't have an eta for a fix right now, I'm sorry for the inconvenience.
